SNIPS/snip-template copy.md

```diff
@@ -0,0 +1,49 @@
+---
+snip: $SNIP_ID
+title: SNIP Purpose and Guidelines
+author: $SNIP_AUTHOR <$SNIP_AUTHOR_EMAIL>
+status: Living
+type: Meta
+created: $SNIP_DATE
+---
+
+## Simple Summary
+
+This snip will deal with the emergency powers the security council will have and how they need to communicate these emergency actions to the broader community.
```

Comment:
I suggest we capitalize SNIP as well as Security Council (instead of security council)

```suggestion
This SNIP will deal with the emergency powers the Security Council will have and how they need to communicate these emergency actions to the broader community.
```